app/src/main/res/values/strings.xml

```diff
@@ -306,60 +306,60 @@
         9. Other memory and crash fixes.
     </string>
 
-    <string name="connection_http3">HTTP3</string>
+    <string name="connection_http3">HTTP/3</string>
 
     <!-- Settings Fragment strings -->
     <string name="settings_ip_network_heading">Choose IP version</string>
     <string name="settings_ip_network_desc">Choose between Internet Protocol versions.</string>
 
     <string name="settings_ptrans_heading">Force IPv4 egress</string>
-    <string name="settings_ptrans_desc">Translate outbound IPv6 traffic to IPv4 on-device. Enable if you are connected to an IPv4-only network.</string>
+    <string name="settings_ptrans_desc">Translate outbound IPv6 traffic to IPv4 on-device. Use this if you are connected to an IPv4-only network.</string>
 
     <string name="settings_allow_bypass_heading">Allow Bypass</string>
-    <string name="settings_allow_bypass_desc">Allow requesting apps to bypass RethinkDNS firewall on-demand. Some audio/video conference apps like Zoom and WhatsApp may require an on-demand bypass to function properly.</string>
-    <string name="settings_lock_down_mode_desc">Cannot allow bypass, or setup proxies when VPN is running in lockdown mode. <u>Go to VPN settings</u>.</string>
+    <string name="settings_allow_bypass_desc">Allow requesting apps to bypass RethinkDNS firewall on-demand. Some audio/video conference apps like the (considered harmful) Zoom and WhatsApp may require an on-demand bypass to function properly.</string>
```

Comment:
no need for the editorial here ;)

Reply:
DFSG guidelines on "steering" the user, and so on. Recommending, etc.